bucket/wasm4.json

```
@@ -0,0 +1,24 @@
+{
+    "version": "2.3.2",
+    "description": "Build retro games using WebAssembly for a fantasy console.",
+    "homepage": "https://wasm4.org/",
+    "license": "ISC License",
```

```suggestion
    "license": "ISC",
```
As per https://spdx.org/licenses/ISC.html